website/ref/unix.md

```diff
@@ -0,0 +1,10 @@
+<!-- toc -->
+
+# Introduction
+
+The `unix:` module provides access to features that only make sense on UNIX
+compatible operating systems such as Linux, FreeBSD, macOS, etc. Do not use
```

Comment:
Instead of saying "do not use", just say that this module does not exist on non-UNIX OSes (assuming you acted on my other comment of not installing the module at all for non-UNIX OSes).